mne_qt_browser/_pg_figure.py

```diff
@@ -1809,34 +1842,86 @@ def __init__(self, main, title="Settings", **kwargs):
-                ch_scaling_layout.addRow(f"{ch} ({self.mne.units[ch]})", ch_spinbox)
+        ch_types_ordered = [
+            ordered_types[idx]
+            for idx in sorted(unique_type_idxs)
+            if ordered_types[idx] in self.mne.unit_scalings.keys()
+        ]
+
+        # Grid layout for channel spinboxes and settings
+        ch_grid_layout = QGridLayout()
+
+        # Create dropdown to choose units
+        self.physical_units_cmbx = QComboBox()
+        self.physical_units_cmbx.addItems(["/ mm", "/ cm", "/ inch"])
+        self.physical_units_cmbx.currentIndexChanged.connect(
+            self._update_sensitivity_spinbox_values
+        )
+        current_units = self.physical_units_cmbx.currentText().split()[-1]
+
+        # Add subgroup box to show channel type scalings
+        ch_scroll_box = QGroupBox("Channel Configuration")
+        ch_scroll_box.setStyleSheet("QGroupBox { font-size: 12pt; }")
+        self.ch_scaling_spinboxes = {}
+        # self.ch_scaling_spinbox_labels = {}
+        self.ch_sensitivity_spinboxes = {}
+        # self.ch_sensitivity_spinbox_labels = {}
+        self.ch_label_widgets = {}
+
+        ch_grid_layout.addWidget(QLabel("Channel Type"), 0, 0)
+        ch_grid_layout.addWidget(QLabel("Scaling"), 0, 1)
+        ch_grid_layout.addWidget(QLabel("Sensitivity"), 0, 2)
+        grid_row = 1
+        for ch_type in ch_types_ordered:
+            self.ch_label_widgets[ch_type] = QLabel(
+                f"{ch_type} ({self.mne.units[ch_type]})"
+            )
+
+            # Make scaling spinbox first
+            ch_scale_spinbox = QDoubleSpinBox()
+            ch_scale_spinbox.setMinimumWidth(100)
+            ch_scale_spinbox.setRange(0, float("inf"))
+            ch_scale_spinbox.setDecimals(1)
+            ch_scale_spinbox.setStepType(QAbstractSpinBox.AdaptiveDecimalStepType)
+            inv_norm = _get_channel_scaling(self, ch_type)
+            ch_scale_spinbox.setValue(inv_norm)
+            ch_scale_spinbox.valueChanged.connect(
+                _methpartial(self._update_spinbox_values, ch_type=ch_type)
+            )
+            self.ch_scaling_spinboxes[ch_type] = ch_scale_spinbox
+
+            # Now make sensitivity spinbox
+            ch_sens_spinbox = QDoubleSpinBox()
+            ch_sens_spinbox.setMinimumWidth(100)
+            ch_sens_spinbox.setRange(0, float("inf"))
+            ch_sens_spinbox.setDecimals(1)
+            ch_sens_spinbox.setStepType(QAbstractSpinBox.AdaptiveDecimalStepType)
+            ch_sens_spinbox.setReadOnly(True)
+            ch_sens_spinbox.setDisabled(True)
+            ch_sens_spinbox.setValue(
+                _calc_chan_type_to_physical(self, ch_type, units=current_units)
+            )
+            self.ch_sensitivity_spinboxes[ch_type] = ch_sens_spinbox
+            # self.ch_sensitivity_spinbox_labels[ch_type] = QLabel(
+            #     f"{ch_type} ({self.mne.units[ch_type]}/{current_units})"
+            # )
```

Comment:
Remove cruft

Reply:
Thanks for doing this. I think's it's ready to be pulled